### PR TITLE
[10.0] Add module nominatim_geolocalize

### PR DESCRIPTION
**Pull request by remi-filament**
This module is derived from base_geolocalize, but replacing Google API by Nominatim / OpenStreetMap one.

### CODE REVIEW

**Comment by rafaelbn** [comment]
======================
Nominatim Geolocalize
======================

Get partner coordinate using Nominatim API

Alternative to Google API

**Comment by pedrobaeza** [comment]
Module name should be `geolocalize_nominatim` for following guidelines.

File: nominatim_geolocalize/README.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Get partner coordinate using Nominatim API
+
+Alternative to Google API
+
+
+
+Credits
+=======
+
+Contributors ------------
```
Comment (pedrobaeza) [inline nominatim_geolocalize/README.rst]: You can use readme by fragments for having this auto-populated. See template module: https://github.com/OCA/maintainer-tools/tree/master/template/module

Comment (remi-filament) [reply nominatim_geolocalize/README.rst]: Thanks @pedrobaeza, I renamed the module as per your suggestion, regarding the README, this is already done, only I kept README.rst for better readability of PR, but I understand this should be replaced by auto-populated one, no ?

Comment (pedrobaeza) [reply nominatim_geolocalize/README.rst]: Yes, sorry for not checking that part.

File: geolocalize_nominatim/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2018 Le Filament (<http://www.le-filament.com>)
```
Comment (NL66278) [inline geolocalize_nominatim/__init__.py]: Please no copyright lines in init files.

File: geolocalize_nominatim/models/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+import urllib2
+
+from odoo import api, fields, models, tools, _
+from odoo.exceptions import UserError
+
+
+def geo_find(addr):
+    if not addr:
+        return None
+    url = 'https://nominatim.openstreetmap.org/search.php/'
```
Comment (NL66278) [inline geolocalize_nominatim/models/res_partner.py]: No need for three assignments. Do one, either using line continuation, or by putting the three partial strings between `(` and `)`.

File: geolocalize_nominatim/views/res_partner.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+        <record id="res_partner_nominatim_form" model="ir.ui.view">
```
Comment (NL66278) [inline geolocalize_nominatim/views/res_partner.xml]: Decrease ident to 4.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Nice addition but needs some changes.

**Comment by NL66278** [comment]
PR seems to be abandoned and is for no longer supported version.

**Comment by remi-filament** [comment]
Thank you @NL66278 for reviewing this PR and making these changes
Sorry I was not able to be more reactive !
We are still using these functionalities in production for some customers that do not want to migrate to newer version, do you think it makes no sense to merge it in 10.0 branch ?
Best Regards

**Comment by NL66278** [comment]
It makes sense. Just push the branch with the requested changes and I will re-open the PR and review again.

**Comment by remi-filament** [comment]
It seems you have already pushed all the requested changes, or am I missing something ?

**Pull request closed without merging**